src/BasisTheory.net.Tests/Tokens/Helpers/TokenFactory.cs

```diff
@@ -77,6 +89,17 @@ public static class TokenFactory
                 f => f.PickRandom(
                     DataRestrictionPolicy.MASK,
                     DataRestrictionPolicy.REDACT));
+        
+        public static readonly Faker<UpdatePrivacyModel> UpdateDataPrivacyFaker = new Faker<UpdatePrivacyModel>()
```

Comment:
`UpdatePrivacyModel` doesn't match the other names. Can we do `PrivacyUpdateModel`?